lib/network/modules/InteractionHandler.js

```diff
@@ -511,7 +511,11 @@ class InteractionHandler {
       else if (event.detail) { /* Mozilla case. */
         // In Mozilla, sign of delta is different than in IE.
         // Also, delta is multiple of 3.
+        // For older FireFox versions
         delta = -event.detail / 3;
+      } //Modern FireFox
```

Comment:
Only Firefox? No other browsers support this?

Also it would look better with a space after `//` and Mozilla doesn't capitalize the second F in Firefox.